packages/toolboxes/toolbox-cosmos/src/cosmosClient.ts

```diff
@@ -57,7 +57,9 @@ export class CosmosClient {
 
   getBalance = async (address: string) => {
     const client = await StargateClient.connect(this.rpcUrl);
-    return client.getAllBalances(address);
+    return client.getAllBalances(address) as unknown as Promise<
```

Comment:
why not using the default return type?
```
import { Coin } from "cosmjs-types/cosmos/base/v1beta1/coin";
```

Reply:
@0x-General TS said it can't interfere this type without cosmjs-types and some other depencency which produced `any` at the final build